pkg/apis/kops/cluster.go

```diff
@@ -316,6 +316,8 @@ type EtcdClusterSpec struct {
 	HeartbeatInterval *metav1.Duration `json:"heartbeatInterval,omitempty"`
 	// Image is the etcd docker image to use. Setting this will ignore the Version specified.
 	Image string `json:"image,omitempty"`
+	// BackupStore is the VFS path where we should backup etcd
+	BackupStore string `json:"backupStore,omitempty"`
```

Comment:
Can we add a container name so we can define the container name? This would allow supporting https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/cluster.go#L184.  Maybe a struct for Etcd backups? I assume will will have settings like frequency, location, container, and maybe more

Reply:
Definitely yes to a struct.  I'm trying to think how best to allow the specification of the container (which we do want) while also avoiding getting users stuck not using the etcd-manager though.

So I suggest we start with hard-coded, and figuring this out (and enabling etcd-manager optionally) is probably next.  Which BTW will allow etcd2 -> etcd upgrades (if you're willing to run bleeding edge stuff)